### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by Dachrono**
- Intall webpack, linters and jest
- Select API
- Select assets
- Create home page, header and footer
- Create content to home page and likes for the items, and add new likes
- Create popup with content, comments and add new comments 
- Create counter to items in home page and for comments in popup
- Create test for the counters

### CODE REVIEW

File: src/test/counterComm.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import counterModule from '../modules/counterComm.js';
+
+describe('Test comment counter', () => {
+  test('Should return 0 when recibe message "whitout comments"', () => {
+    const message = 'whitout comments';
+    const result = counterModule(message);
+    expect(result).toBe(0);
+  });
```
Comment (TadesseDev) [inline src/test/counterComm.test.js:8]: - [ ] Good Job adding the tests team, But please add a test for the like counter. 
- [ ] Also, we need to have tests for all edge cases. This includes, `0`, `1`, and >`1` comments, like, or items, Please apply this test for all counter functions.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi Team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅
 - [x]  Correct GitFlow  ✅
 - [x]  Good commit history  ✅
 - [x]  Good PR title/description  ✅
 - [x]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/crudInvolvementAPI.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const axios = require('axios').default;
+
+const appId = '19ZJvuqHVwq8iUHpGDJZ';
```
[the remark this answers is not in the document]
Comment (Dachrono) [reply src/modules/crudInvolvementAPI.js:3]: Solved

File: src/test/counterComm.test.js (hunk 1 above)
Comment (Dachrono) [reply src/test/counterComm.test.js:8]: We create a test for all conter functions

Comment (Dachrono) [reply src/test/counterComm.test.js:8]: solved

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 
Hi @Dachrono & @lRebornsl, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good UI design  :white_check_mark: 
- [x] Application works as expected  :white_check_mark:  
- [x] All unit tests work as expected  :white_check_mark:  
- [x] Git flow is used correctly :white_check_mark: 
- [x] Repository has a professional readme file :white_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
